x-pack/plugins/ml/public/application/components/anomalies_table/anomaly_details_utils.tsx

```diff
@@ -488,6 +488,31 @@ export const AnomalyExplanationDetails: FC<{ anomaly: AnomaliesTableRecord }> =
       description: explanation.high_variance_penalty ? yes : no,
     });
   }
+  if (explanation.multimodal_distribution !== undefined) {
+    impactDetails.push({
+      title: (
+        <EuiToolTip
+          position="left"
+          content={i18n.translate(
+            'xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.multimodalTooltip',
+            {
+              defaultMessage:
+                'Indicates whether the prior distribution of the observed time series is multi-modal or has a single mode',
```

Comment:
Does 'observed' add an important detail here? If not, I think we can remove it from the sentence.
```suggestion
                'Indicates whether the prior distribution of the time series is multi-modal or has a single mode.',
```